core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -103,4 +103,6 @@ private TableProperties() {}
 
   public static final String WRITE_TARGET_FILE_SIZE_BYTES = "write.target-file-size-bytes";
   public static final long WRITE_TARGET_FILE_SIZE_BYTES_DEFAULT = Long.MAX_VALUE;
+
+  public static final boolean CHECK_NULLABILITY_DEFAULT = true;
```

Comment:
Can you remove this? It isn't a table property so it doesn't make sense to have it here anymore.

Reply:
Ok. Will use concrete boolean value.